BreanaMarie/tic-tac-toe/ticTacToe.js

```diff
@@ -0,0 +1,458 @@
+console.log ("connected");
+// wait for the DOM to finish loading
+$(document).ready(function(){
+  // all code to manipulate the DOM
+  // goes inside this function
+
+
+
+var round='';
+var i=0;
+
+//turn counter x = even o = odd 
+//declare draw
+function turn(blah){
```

Comment:
Your spacing throughout this file makes it a pleasure to read.  Thank you :rabbit: